### PR TITLE
Update CI report format to csv

### PR DESCRIPTION
**Pull request by chuneuny-emily**
### Summary
Changed the CI test report format from xml to user friendly csv format.

### Checklist

- [ ] I have added unit tests to cover my changes.​
- [ ] I have added integration tests to cover my changes.​
- [ ] I have added the description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md).​
- [ ] I have updated the [documentation](https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly

### License

- [ ] I submit _my code changes_ under the same [MIT License](https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below).

```python
# Copyright (C) 2023 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by chuneuny-emily** [comment]
daily test results: https://github.com/openvinotoolkit/datumaro/actions/runs/6105654969

File: tox.ini [hunk 1]
```diff
@@ -2,6 +2,8 @@
 isolated_build = true
 skip_missing_interpreters = true
 
+[pytest]
+addopts = --csv=.tox/tests-{env:TOXENV_OS}-{env:TOXENV_PYVER}.csv
```
Comment (vinnamkim) [inline tox.ini]: Doesn't it is duplicated with L75, 80, and 90?

Comment (chuneuny-emily) [reply tox.ini]: Yes, you are correct. Lines 5 and 6 of the configuration file are duplicated with lines 75, 80, and 90. These lines define the same [pytest] section and specify the same addopts option with different values. I'll remove lines 5 and 6.

**Review by vinnamkim — APPROVED** [review]
LGTM.

**Pull request merged**